students/ABartles/Exercise3/Mailroom.py

```diff
@@ -0,0 +1,82 @@
+def Send_Thanks(dnrslst):
+    response = "Not a donor or the word list"
+
+    while response not in dnrslst:
+        print("----------------------------------------")
+        response = input("Who would you like to thank ('first last', 'list', or 'exit'? ")
+        response = response.title()
+
+        if response == "Exit":
+            break
+        elif response == "List":
+            print("----------------------------------------")
+            print("Below is the list of Donors:")
+            print(List_Donors(dnrslst))
+        elif response != "List":
+            print("----------------------------------------")
+            amount = input("How much was dontated by {}? ".format(response))
+            dnrslst = Rec_Donation(dnrslst, response, amount)
+            print("----------------------------------------")
+            Thanks_Email(response, dnrslst)
+
+def List_Donors(dnrslst):
+    name_lst = []
+    for i in dnrslst:
+        if i[0] not in name_lst:
+            name_lst.append(i[0])
+    return name_lst
+
+def Rec_Donation(dnrslst, donor, money):
+    donor_tpl = (donor, money)
+    dnrslst.append(donor_tpl)
+    return dnrslst
+
+def Thanks_Email(donor, dnrslst):
+    money = Sum_Gift(donor, dnrslst)
+    print("Dear {},".format(donor))
+    print("Thank you for your generous gift of {}.".format(money))
+
+def Sum_Gift(donor, dnrslst):
+    total = 0
+    for i in dnrslst:
+        if i[0] == donor:
+            total = total + int(i[1])
+    return total
+
```

Comment:
You might consider using a different data structure, something like 
```
donors = [['Donor', [120, 250]], ['Donor2', [120, 250]],]
```
this would simplify some of your code for dealing with donations and let you use Python built-ins like `sum()` and `len()` on the donation lists to get the total and count of donations.